posthog/models/organization.py

```diff
@@ -1,15 +1,33 @@
 import uuid as uuidlib
 from enum import IntEnum
 from multiprocessing import Value
-from typing import Any, Optional, Tuple
+from typing import Any, Dict, Optional, Tuple
 
 from django.conf import settings
-from django.db import models
+from django.db import models, transaction
 from django.dispatch import receiver
 
 from .utils import UUIDModel, sane_repr
 
 
+class OrganizationManager(models.Manager):
+    def bootstrap(
+        self, user: Any, *, team_fields: Optional[Dict[str, Any]] = None, **kwargs
+    ) -> Tuple["Organization", "OrganizationMembership", Any]:
+        from .team import Team
+
+        with transaction.atomic():
+            organization = Organization.objects.create(**kwargs)
+            organization_membership = OrganizationMembership.objects.create(
+                organization=organization, user=user, level=OrganizationMembership.Level.ADMIN
+            )
+            team = Team.objects.create(organization=organization, **(team_fields or {}))
+            user.current_organization = organization
+            user.current_team = team
```

Comment:
I'm not sure about the `current_team` & `current_organization` approach of saving this stuff in the DB, seems prone to discrepancies (e.g. current team not matching current org), plus a lot of extra changes to the DB. I would consider just sending an extra header on each request indicating which team is selected on the frontend. This also makes sense if you're operating on two devices, you could easily be working with different teams on each device (e.g. I would open an incognito window if I wanted to see website numbers against app numbers).

Reply:
I agree that that would be great, but it would also be a significantly larger refactor (consider insights, which would need to be project-nested then like so: `/api/projects/<project_id>/insights/` – this would have to apply to everything). Could we do this a bit later in another PR? One of the points here is that this one is already too big (which is also true).

Reply:
I was considering something easier, just add a header (e.g. `Posthog-Project`), on the backend it's a simple mixin to parse that and on the frontend I believe you can use the `api` lib that all requests use (or an interceptor). I do think this might simplify the current PR, but obviously not dead set against doing it this way, we can definitely consider this for an improvement later on.